Compiler/BackEnd/BackendDAEUtil.mo

```diff
@@ -7470,6 +7469,7 @@ algorithm
     // handle special flags, which disable modules
     disabledModules := deprecatedDebugFlag(Flags.NO_PARTITIONING, disabledModules, "clockPartitioning", "preOptModules-");
     disabledModules := deprecatedDebugFlag(Flags.DISABLE_COMSUBEXP, disabledModules, "comSubExp", "preOptModules-");
+    disabledModules := deprecatedDebugFlag(Flags.RESOLVE_LOOPS, enabledModules, "resolveLoops", "preOptModules+");
```

Comment:
Shouldn't this disable `resolveLoops`?
`disabledModules := deprecatedDebugFlag(Flags.RESOLVE_LOOPS, disabledModules, "resolveLoops", "preOptModules-");`